bgc_data/phytoinfo.dataresource.yaml

```diff
@@ -22,6 +24,10 @@ schema:
     title: ""
     type: number
     required: false
+  - name: FUNCTIONALGROUP
+    title: ""
```

Comment:
These title fields could be set to a short description of the field, especially for fields like this where the name itself doesn't quite make it obvious (at least to me...).

Otherwise, titles are not required, so if no title is provided for a field it can just be left out all together to make the file a bit easier to read.